packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -935,7 +935,15 @@ function isOverTarget(point: EventPoint, target: Element) {
 
 function shouldPreventDefaultDown(target: Element) {
   // We cannot prevent default if the target is a draggable element.
-  return !(target instanceof HTMLElement) || !target.hasAttribute('draggable');
+  if (!(target instanceof HTMLElement)) {
+    return true;
+  }
+
+  if (target.tagName === 'A' && target.getAttribute('draggable') !== 'false') {
+    return false;
+  }
```

Comment:
if target is `a` tag and doesn't have `draggable="false"` attr, we shouldn't prevent default to enable to drag the link